ui/nuxeo-document-preview.js

```diff
@@ -282,12 +282,14 @@ import './viewers/nuxeo-video-viewer.js';
       ) {
         const filteredViews = this.document.properties['picture:views'].filter((view) => view.title === 'FullHD');
         if (filteredViews.length > 0) {
-          return `${filteredViews[0].content.viewUrl}`;
+          return `${
+            filteredViews[0].content.viewUrl ? filteredViews[0].content.viewUrl : filteredViews[0].content.data
```

Comment:
Remove repititive code . have const for filteredViews[0].content

Reply:
done